### PR TITLE
Magento theme foundations

### PR DESCRIPTION
**Pull request by mostro20**
This is new theme foundation. There are still some hardcoded colours from Yume to be dealt with, as well as updating the header section once we've got something styled up on the website and research front-end. But nevertheless, this is a much better foundation then the previous build and has the essence of the theme.

### CODE REVIEW

File: Magento_Theme/templates/html/rwahsnav.phtml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Copyright © 2015 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+?>
+<div class="rwahsglobalnav">
+    <ul class="nav nav-tabs">
+        <li role="presentation"><a href="https://uat-www.histwest.org.au">Website</a></li>
```
Comment (leftclickben) [inline Magento_Theme/templates/html/rwahsnav.phtml]: These should be production links, not UAT links.  There's a risk that we'll forget to change it, and I'd rather have it incorrectly point to production from UAT, than incorrectly point to UAT from production.

**Comment by leftclickben** [comment]
Just the one issue about the cross-site links.

Also, there seems to be a lot of code coming out in this PR, and not much going back in to replace it, is that correct?

**Comment by mostro20** [comment]
Have updated the Nav links - good point. RE: removal of code: sadly, yes, the mass removal of code is correct. Almost everything that is coming out is being replaced with the fact that I worked out how to use reference containers properly. Do note, that in the next PR, when I have to replace the hardcoded Yume attributes I expect a bit more code to come back in. But it's actually a nice PR right now as it's core / base of what we need.

**Comment by leftclickben** [comment]
I just noticed that the link to the research frontend is "Collections" bit the domain name is `research` so I've been thinking of it as having the label "Research" (and that's what it is in the research frontend). We need a consensus on this so it's consistent.

**Pull request merged**